presidio-anonymizer/presidio_anonymizer/entities/engine/result/operator_result.py

```diff
@@ -0,0 +1,23 @@
+class OperatorResult:
+    """A class to hold data for engines results either anonymize or deanonymize."""
+
+    def __init__(self, text: str, operator_name: str, start: int, end: int,
```

Comment:
This can also own a PII entity instead of duplicating its fields

Reply:
I need to contemplate on this one... not so sure.
The PIIEntity is currently being used for the request.
This item is a return object. Using it for both, will chain the code so in the future it will be harder to change it.